charts/tempo-distributed/values.yaml

```diff
@@ -241,6 +241,7 @@ querier:
 # Configuration for the query-frontend
 queryFrontend:
   query:
+    enabled: true # required for grafana version <7.5 for compatibility with jaeger-ui. Doesn't work on ARM arch
```

Comment:
Linting is complaining about this, I think you have to add at least two spaces before a comment.

Reply:
done